.ci/env/openblas.sh

```diff
@@ -46,23 +76,49 @@ done
 target=${target:-ARMV8}
 host_compiler=${host_compiler:-gcc}
 compiler=${compiler:-aarch64-linux-gnu-gcc}
-cflags=${cflags:--march=armv8-a+sve}
+
+target_arch=${target_arch:-$(uname -m)}
+blas_prefix="${ONEDAL_DIR}/__deps/openblas_${target_arch}"
 
 sudo apt-get update
 sudo apt-get -y install build-essential gcc gfortran
-git clone https://github.com/xianyi/OpenBLAS.git
-CoreCount=$(lscpu -p | grep -Ev '^#' | wc -l)
-pushd OpenBLAS
-  make clean
+blas_src_dir=${blas_src_dir:-$OPENBLAS_DEFAULT_SOURCE_DIR}
+if [[ ! -d "${blas_src_dir}" ]] ; then
+  git clone https://github.com/OpenMathLib/OpenBLAS "${blas_src_dir}"
```

Comment:
Can we also do a shallow clone as well?

Reply:
Yeah, good idea. As part of this, I've added options to pass in the version to check out, as well as a prefix variable. In future commits, I want to split the CI out into building TBB and OpenBLAS themselves, so that we can cache them. We don't need to build OpenBLAS and TBB every time we build

Reply:
With bazel builds in future this also can be cashed within bazel itself